repository/j.json

```diff
@@ -788,6 +788,17 @@
 			]
 		},
 		{
+			"name": "jsFiddle Syntax Theme",
+			"details": "https://github.com/HiDAl/SublimeText-jsFiddle-Syntax",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```

Comment:
We no longer allow branch-based releases. See https://packagecontrol.io/docs/submitting_a_package#Step_4 for more info.